### PR TITLE
replace mongo key literal with environmental variable

### PR DESCRIPTION
**Pull request by Spellbreeze**
Collaborator(s): Xander

### CODE REVIEW

**Review by keithtkj — COMMENTED** [review]
looks good

File: backend/server.js [hunk 1]
```diff
@@ -11,15 +11,14 @@ if (result.error) {
...
-
+const app = express()
+const PORT = process.env.PORT
+const MONGO_KEY = process.env.MONGO_KEY
 
 app.use(cors())
 
 // mongoose.connect('mongodb://127.0.0.1:27017/CivilityOptics', { useNewUrlParser: true });
-mongoose.connect('mongodb+srv://CivilityOptics:Dontpush2134@civility1.7zzflo5.mongodb.net/?retryWrites=true&w=majority', { useNewUrlParser: true })
+mongoose.connect(MONGO_KEY, { useNewUrlParser: true })
```
Comment (madhura112) [inline backend/server.js:21]: looks good!

**Review by madhura112 — APPROVED** [review]

**Review by keithtkj — APPROVED** [review]

File: backend/server.js (hunk 1 above)
Comment (parkaransingh) [reply backend/server.js:21]: ok good!

File: backend/server.js [hunk 2]
```diff
@@ -11,15 +11,14 @@ if (result.error) {
   throw result.error
 }
 
-const app = express();
-const PORT = process.env.PORT;
-
-
```
Comment (menelik-g) [inline backend/server.js:17]: Good removals

**Review by parkaransingh — APPROVED** [review]
I approve

**Review by menelik-g — APPROVED** [review]

**Pull request merged**